client/my-sites/stats/stats-notices/style.scss

```
@@ -9,4 +9,23 @@
 		vertical-align: middle;
 		display: inline-block;
 	}
+
+	&.inner-notice-container--emerald {
+		p,
+		button,
+		a {
+			/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+			font-size: 14px;
```

```suggestion
			font-size: $font-body-small;
```
Could we use the variable for the font size?